crates/bootstrap_srv/src/http.rs

```diff
@@ -190,6 +190,8 @@ fn tokio_thread(
             let shutdown_handle = axum_server::Handle::new();
 
             for addr in server_config.addrs {
+                tracing::info!("Binding to: {}", addr);
```

Comment:
Had some trouble here. We were binding to `0.0.0.0:443` and `[::]:443` but with dual stack available, binding to `[::]:443` hangs after we've successfully bound to `0.0.0.0:443`. 

I can remove these debug logs but a little extra debug help probably doesn't hurt